hooks/useUserStateSyncedWithCookies.tsx

```diff
@@ -8,42 +8,48 @@ import {
 } from '../helpers/_cookies'
 import Cookies from 'js-cookie'
 
+type CookieMap = { [cookieName: string]: string }
```

Comment:
I'm not so sure i get this syntax, can you explain to me pls?
Is like saying:
each cookieName which is a string, is a string ?? 
or like: COOKIE_NAME_SEARCH_COUNT: string ?

Reply:
Yes, an object with strings as keys and strings as values. E.g. would fail if you put a number as a key or a value.

Reply:
The actual cookie names we have defined in the constants are not taken into account in this type. This could be an improvement.

Reply:
ahh ok yep make sense now.

yeah would be nice to have the actual name. Next refactor ;)